source/backend/tests/unit/api/Services/CompensationRequisitionServiceTest.cs

```diff
@@ -5,6 +5,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using FluentAssertions;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
```

Comment:
I've never seen this import on our unit tests before. Are we sure it is needed here?

Reply:
removed